build/Dockerfile-go1.20.6-solc0.8.13-ubuntu

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:20.04 as solc_0.8.13_builder
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Asia/Seoul
+RUN apt update
+RUN apt install -yq tzdata && \
+    ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
+    dpkg-reconfigure -f noninteractive tzdata
+RUN apt install -y git lsb-core sudo libboost-all-dev build-essential cmake z3
+RUN git clone --depth 1 --recursive -b v0.8.13 https://github.com/ethereum/solidity
+RUN cd /solidity && cmake -DCMAKE_BUILD_TYPE=Release -DTESTS=0 -DSTATIC_LINKING=1
+RUN cd /solidity && touch prerelease.txt
+RUN cd /solidity && make solc
+RUN cd /solidity && install -s solc/solc /usr/bin
+
+FROM ubuntu:20.04 as go_builder
+RUN apt update && apt install -y software-properties-common
+RUN add-apt-repository ppa:longsleep/golang-backports -y && \
+    apt install -yq tzdata && \
+    ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
+    dpkg-reconfigure -f noninteractive tzdata
+RUN apt install -y build-essential bash gcc musl-dev openssl wget golang golang-1.17
+RUN wget -O go.src.tar.gz https://dl.google.com/go/go1.20.6.src.tar.gz
+RUN tar -C /usr/local -xzf go.src.tar.gz
+RUN cd /usr/local/go/src/ && \
+    ./make.bash
```

Comment:
Is it a required step (`./make.bash`)? this is because the typical installation steps seem not to involve running this script.

Reply:
Running `make.bash` is a typical steps in [installation from source](https://go.dev/doc/install/source)

Reply:
I overlooked that the downloaded were source files, not the binaries. Thanks.